app/locale/de-CH.coffee

```diff
@@ -1407,7 +1407,7 @@ module.exports = nativeDescription: "Dütsch (Schwiiz)", englishDescription: "Ge
 #    delete_clan: "Delete Clan"
 #    leave_clan: "Leave Clan"
 #    join_clan: "Join Clan"
-#    invite_1: "Invite:"
+#    invite_1: "Iilade:"
```

Comment:
I keep seeing "contribution" as this translation. Do you have a good site where I can look up Schwiiz?